docs/docs/topics/schema.mdx

```diff
@@ -197,120 +282,127 @@ nodes:
-  - name: Person
-    namespace: Example
+```yaml
+generics:
+  - name: GenericDevice
+    namespace: Infra
+    icon: "mdi:router"
+    default_filter: "name__value"
     attributes:
       - name: name
         kind: Text
         unique: true
-  - name: Car
-    namespace: Example
-    uniqueness_constraints: ["owner", "model__value"]
+        order_weight: 1000
+
+  - name: Asset
+    namespace: Infra
+    icon: "mdi:server"
     attributes:
-      - name: model
+      - name: asset_tag
+        label: Asset Tag
         kind: Text
-    relationships:
-      - name: owner
-        peer: ExamplePerson
-        optional: false
-        cardinality: one
-```
-
-### Human friendly identifier (hfid)
-
-In addition to the internal id automatically generated for each object (UUID), Infrahub provide a way to define a set of fields that will constitue a human friendly identifier (hfid).
-
-:::success Use Case
-
-The `hfid` is meant to be used to reference objects uniquely across systems, or even before an object has been created in Infrahub.
-Having a human friendly way to identify an object is very important to build robust data synchronization between systems and to provide true idempotent script.
-
-In the network industry:
-
-- the `hfid` of a device will be could be its `name` : `atl1-edge01`
-- the `hfid` of an interface will be the combination of the name of the device and the name of the interface: (`atl1-edge01`, `Ethernet1`)
-
-:::
-
-In the example below, each `ExamplePerson`,  will have a `hfid` composed of his/her `lastname` and `firstname`.
-
-`human_friendly_id` can be composed of N number of attribute or relationship of cardinality one:
-
-- For an attribute, the valid format is `<attributename>__value`. *Currently only value is supported but in the future the plan is to allow additional metadata to be used as well.*
-- For a relationship, the name of the relationship and the name of a unique attribute must be provided `<relationshipname>__<attributename>__value`. *Only relationship of cardinality one are supported and the relationship must be mandatory.*
+        unique: true
+        order_weight: 1000
 
-```yaml {4} showLineNumbers
 nodes:
-  - name: Person
-    namespace: Example
-    human_friendly_id: ["lastname__value", "firstname__value"]
-    attributes:
-      - name: firstname
-        kind: Text
-      - name: lastname
-        kind: Text
+  - name: Device
+    namespace: Infra
+    inherit_from:
+      - "InfraGenericDevice"
+      - "InfraAsset"
 ```
 
-:::info
+### Inherited properties
 
-If a `human_friendly_id` is defined on a node but no `uniqueness_constraints` are present, a uniqueness constraints matching the `human_friendly_id` will be automatically created.
+#### List of inheritable properties
 
-If at least one attribute is marked as unique, and `human_friendly_id` hasn't been defined, `human_friendly_id` will be automatically created.
+- **`human_friendly_id`**: A user-friendly identifier for the entity.
```

Comment:
I would not make them bold and in a code block. It looks a bit weird in the rendered docs to me.
Not a blocking thing though